build.gradle

```diff
@@ -59,8 +59,8 @@ dependencies {
     compile group: 'commons-cli', name: 'commons-cli', version: '1.2'
     compile group: 'org.apache.commons', name: 'commons-text', version: '1.1'
     compile 'com.google.guava:guava:23.0'
-    // compile 'com.github.eilslabs:RoddyToolLib:develop-SNAPSHOT'
```

Comment:
We should make sure (somehow) that this is changed exactly before releases. Any ideas?

Reply:
This is an old release but you are right. This should be fixed.

Reply:
Not sure whether my message was clear. I just was thinking of a way to prevent that this is changed back and forth on development branches and the master branch.